x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowTaskCleaner.java

```diff
@@ -63,9 +64,9 @@ public void clusterChanged(final ClusterChangedEvent event) {
                 // the index exists, do not clean this persistent task
                 continue;
             }
-            IndexNotFoundException e = new IndexNotFoundException(followerIndex);
+            IndexNotFoundException infe = new IndexNotFoundException(followerIndex);
```

Comment:
This name change is to avoid confusion with the name of the `Exception` instance in the `onFailure` callback.